src/features/transforms/Transforms.js

```diff
@@ -30,39 +34,78 @@ const mapDispatch = (dispatch, ownProps) => {
     changeLoops: (event) => {
       dispatch(setNumLoops(event.target.value));
     },
+    onSizeChange: (event) => {
+      dispatch(setShapeStartingSize(event.target.value));
+    },
+    onOffsetXChange: (event) => {
+      dispatch(setXFormOffsetX(event.target.value));
+    },
+    onOffsetYChange: (event) => {
+      dispatch(setXFormOffsetY(event.target.value));
+    },
   }
 }
 
 class Transforms extends Component {
   render() {
     return (
       <div className="transforms">
-        <Panel className="shapes-panel">
-          <h4>Input Shapes</h4>
+        <Card className="p-3">
+          <h4>Select a shape</h4>
           <ShapeList />
-        </Panel>
-        <Panel className="transforms-panel">
+        </Card>
+
+        <Card className="mt-3 p-3">
           <h4>Modifiers</h4>
-          <Panel className="options-panel">
-            <Form horizontal>
-              <FormGroup controlId="loop-count">
-                <Col componentClass={ControlLabel} sm={4}>
-                  Number of Loops
-                </Col>
-                <Col sm={8}>
-                  <FormControl type="number" value={this.props.loops} onChange={this.props.changeLoops} onKeyDown={disableEnter}/>
-                </Col>
-              </FormGroup>
-            </Form>
-          </Panel>
-          <ListGroup>
+          <Row className="align-items-center pt-3 pb-2">
+            <Col sm={4}>
+              <Form.Label htmlFor="shape-size">
+                Starting size
+              </Form.Label>
+            </Col>
+
+            <Col sm={8}>
+              <FormControl id="shape-size" type="number" value={this.props.starting_size} onChange={this.props.onSizeChange} onKeyDown={disableEnter} />
+            </Col>
+          </Row>
+
+          <Row className="align-items-center pb-2">
+            <Col sm={4}>
+              <Form.Label htmlFor="shape-offset">
+              Offset
+              </Form.Label>
+            </Col>
+
+            <Col sm={8}>
```

Comment:
Something is funny with these items. They stretch over the right side when the screen is small (like on mobile). I can affect it a little by changing the sm, but that doesn't seem to be the right fix.

Reply:
![screenshot-2020-03-18-1584570666](https://user-images.githubusercontent.com/1812882/77013594-f43c5800-6935-11ea-8e8b-892b6fff3670.png)